src/main/java/de/tum/in/www1/artemis/config/ApiVersionFilter.java

```diff
@@ -5,16 +5,16 @@
 
 import java.io.IOException;
 
-import javax.servlet.*;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Profile;
 import org.springframework.stereotype.Component;
 
+import jakarta.servlet.*;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
```

Comment:
This PR contains a lot of import-order changes like these. Are these on purpose?

Reply:
`artemis-spotless.importorder` probably needs to be adapted if the renamed dependencies should be placed in their own imports section like before.

Reply:
Currently, `javax` has its own special case here. It would be interesting to know if this was due to the similar name to `java` or if these imports should generally stay on top. In the case of 2) we should probably adapt the file.
![grafik](https://user-images.githubusercontent.com/26540346/204103046-f61abb0c-a550-40c1-84f6-2ba13cb6a0dc.png)